docs/changelog/4_x.rst

```diff
@@ -670,3 +670,45 @@ Bug fixes
 
 * Make overloading a link when omitting ``link`` keyword work
   (:eql:gh:`#6718`)
+
+4.6
+===
+
+* Add Discord OAuth provider
+  (:eql:gh:`#6708`)
+
+* Add Slack OpenID Connect provider
+  (:eql:gh:`#6720`)
```

Comment:
Wait, are we backporting these to 4.x?? They're changing the config schema.

Reply:
Given the feedback, I'll remove them from the backport.